bob/Cargo.toml

```diff
@@ -29,6 +32,7 @@ lazy_static = "1.4.0"
 libc = "0.2"
 log = "0.4"
 metrics = { version = "0.17", features = ["std"] }
+mimalloc = { version = "*", default-features = false, optional = true }
```

Comment:
Don't use `*` in version. Please specify `0.1` to avoid errors due to loss of backward compatibility

Reply:
You should also limit this dependency to only be enabled on `musl` + `x64`

Reply:
> Don't use `*` in version. Please specify `0.1` to avoid errors due to loss of backward compatibility

Done.